### PR TITLE
Update fish.desktop

### PR DESCRIPTION
**Pull request by scardracs**
Update .desktop file (fish/Fish and added comment)

## Description

Updated Name and added a comment for DEs like Plasma

### CODE REVIEW

**Comment by floam** [comment]
Since we are touching it: is there a specific reason this file needs to be in the root?

**Comment by scardracs** [comment]
> Since we are touching it: is there a specific reason this file needs to be in the root?

No, it's not really needed

**Comment by krobelus** [comment]
> Since we are touching it: is there a specific reason this file needs to be in the root?

Yeah I think it's safe to move it to `share/applications` (that shouldn't break distributors since our build system installs it) but that can be a follow-up.

File: fish.desktop [hunk 1]
```diff
@@ -5,7 +5,8 @@ Terminal=true
 # This file is for use in appimages, which rewrite the path.
 # Other uses would have to replace these with the correct path.
 Exec=fish
-Name=fish
+Name=Fish
```
Comment (krobelus) [inline fish.desktop:8]: I don't think we use uppercase fish anywhere

Comment (floam) [reply fish.desktop:8]: Also looking for a reasoning for the capital F fish.

Comment (scardracs) [reply fish.desktop]: It was just more good-looking (at least on Plasma)

**Review by floam — APPROVED** [review]
LGTM

**Pull request merged**